rotkehlchen/db/snapshots.py

```diff
@@ -172,21 +172,21 @@ def _export(
         It then writes the serialized data to a csv file.
         """
         with self.db.conn.read_ctx() as cursor:
-            display_date_in_localtime = self.db.get_settings(cursor).display_date_in_localtime
+            settings = self.db.get_settings(cursor)
```

Comment:
instead of loading all the settings into memory can you read the two that you need from `CachedSettings`. The reason is that `get_settings` executes `db_settings_from_dict` that has a huge logic that you really don't need

Reply:
The catched settings get updated in `set_setting` when settings change